lms/djangoapps/certificates/api.py

```diff
@@ -63,6 +63,7 @@ def format_certificate_for_user(username, cert):
         "grade": cert.grade,
         "created": cert.created_date,
         "modified": cert.modified_date,
+        "passing": is_passing_status(cert.status),
```

Comment:
is this "passing" or "passed"?  I also wonder if the name should be "is_passing" (or "is_passed")